CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to Boxwise
+
+First off: **thank you**. You will be helping build software that distributes clothes and food to thousands of refugees and other people in need. It gives comfort and dignity to people in a vulnerable situation. That can't be said for most open source contributions.
+
+**If you want to join and support us**, just [write us at hello@boxwise.co](mailto:hello@boxwise.co) and include your github handle and / or CV!
+
+**Do you need help using Boxwise, or you found a bug, or got an idea for a feature?** [Please write us at helpme@boxwise.co!](mailto:helpme@boxwise.co)
+
+### How do we work in general?
+
+Quick summary:
+- We follow [Agile methods](https://medium.com/@svharivinod/rolling-out-features-for-zomato-using-agile-8f24bba1ddd8) ([short video](https://www.youtube.com/watch?v=QCif_-r8eK4)) to roll out new features. 
+- Our sprint cycle is about a month. In this month we meet twice online - once in the middle of the month to check-in and discuss problems, once at the end/beginning of the month to demo our progress of the sprint. We ask you to join at least the demo call once per month.
+- We use slack as our main communication tool and do asynchronous [stand-ups](https://en.wikipedia.org/wiki/Stand-up_meeting) there once a week.
+- On Fridays, we offer a space for troubleshooting if you are stuck and need to dicuss something about your task.
+
+### How do I get my tasks? Where do I find information about them?
+We are using trello for Task Management. Our [product manager](https://www.youtube.com/watch?v=502ILHjX9EE) assigns the tickets to everybody taking part in a sprint. Each ticket should include a user story to understand why we need the ticket. If you are missing information, do not feel shy to write in slack.
+
+### How do I start with my task?
+We would like to do [test driven development](https://www.browserstack.com/guide/tdd-vs-bdd-vs-atdd) where possible. Thinking about the tests first forces you to think about a clear image how a screen or endpoint must look like.
+Besides that, check our [guidelines](#guidelines) and [development principles](#development-principles).
+
+### I'm stuck. How do I get help?
+Basically write in the #dev channel in slack or come to the troubleshooting meeting on Friday mentioned above.
```

Comment:
"Write in the dev channel first, and ask for support head of the friday troubleshooting meeting if needed"

we agreed that people shouldn't just show up on the troubleshooting as they may not find the right people, they should comment in  #dev first.

Reply:
changed.